build/vsts-helpers.ps1

```diff
@@ -13,46 +12,27 @@
 		$Colour
 	)
 	
-	# Store the colour to be able to revert back to it.
-	$previousColour = [System.Console]::ForegroundColor
-	
-	# Set the appropriate foreground text colour.
-	switch ($Colour)
-	{
-		"Red" 		{ [System.Console]::ForegroundColor = [System.ConsoleColor]::Red }
-		"Green" 	{ [System.Console]::ForegroundColor = [System.ConsoleColor]::Green }
-		"Blue" 		{ [System.Console]::ForegroundColor = [System.ConsoleColor]::Blue }
-		"Cyan" 		{ [System.Console]::ForegroundColor = [System.ConsoleColor]::Cyan }
-		"Magenta" 	{ [System.Console]::ForegroundColor = [System.ConsoleColor]::Magenta }
-		"Yellow" 	{ [System.Console]::ForegroundColor = [System.ConsoleColor]::Yellow }
-		"Bold" 		{ [System.Console]::ForegroundColor = [System.ConsoleColor]::White }
-		"Black" 	{ [System.Console]::ForegroundColor = [System.ConsoleColor]::Black }
-	}
+	# Initialize the variable to output.
+	$HelpMessage = ''
 	
 	# Add some extra spacing to either side of the message.
 	$length = $Message.Length + 2
 	
-	# Write the top border.
-	Write-Host "+" -NoNewline
-	for ($i = 0; $i -lt $length; $i++)
-	{
-		Write-Host "-" -NoNewline
+	function WriteBorder($remark){
```

Comment:
Sorry to be pedantic, but could the function `{` opening brace be put on a new line? I really wish there was a proper powershell formatter (like clang-format for cpp), but unfortunately that isn't the case and the vscode extension is quite limited in that regard.

Reply:
This is entirely the result of automatic formatting, which happens when you press Shift+Alt+F in VSCode, so it's not my intentional choice,I didn't do any configuration. It seems that this is the preference of vscode - PowerShell team.

But it's also in line with my habits, because my first language is Java, which is what the official cases are all about.

Reply:
Ok, don't worry about it. It's not a big deal.